datasets/npr/verkooppunten/v0.0.1.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "verkooppunten",
+  "type": "table",
+  "version": "0.0.1",
+  "provenance": "Extractie van RDW, van geografische data van verkooppunten: 'https://opendata.rdw.nl/resource/fk68-nf2y.json' en 'https://opendata.rdw.nl/resource/cgqw-pfbp.json'",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "identifier": "verkooppuntNummer",
+    "required": [
+      "schema",
+      "verkooppuntNummer"
+    ],
+    "display": "verkooppuntNummer",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "verkooppuntNummer": {
+        "type": "string",
+        "provenance": "SellingPointNumber",
+        "description": "Unieke identificatie van een verkooppunt."
+      },
+      "verkooppuntId": {
+        "type": "integer",
+        "provenance": "SellingPointId",
+        "description": "Niet-unieke identificatie van een verkooppunt, zoals afgesproken tussen providers. De code is numeriek (ivm gegevensinvoer telefoon). De nummers zijn maximaal 6-cijferig (vb 9312, 13199), maar kunnen in de toekomst langer worden. Het behoeven niet altijd parkeerautomaat-identificaties te zijn: ook losse nummers op bebording komen voor. De nummers zijn niet uniek: eenzelfde verkooppuntId kan op meer automaten of borden zijn weergegeven, die dan een verkooppuntgroep vormen."
+      },
+      "verkooppuntOmschrijving": {
+        "type": "string",
+        "provenance": "sellingpointdesc",
+        "description": "Omschrijving van een verkooppunt."
+      },
+      "beginDatumVerkooppunt": {
+        "type": "string",
+        "provenance": "startdatesellingpoint",
+        "description": "Datum vanaf wanneer een bepaald verkooppuntId actief is."
+      },
+      "eindDatumVerkooppunt": {
+        "type": "string",
+        "provenance": "enddatesellingpoint",
+        "description": "Datum waarop een bepaald verkooppuntNummer niet meer actief is. Gelijk aan 29991231 wanneer verkooppuntNummer momenteel nog actief is."
+      },
+      "versieIngangDatum": {
+        "type": "string",
+        "provenance": "",
+        "description": "Datum vanaf wanneer een bepaald verkooppuntNummer actief is. Gelijk aan eindDatumVerkooppunt van voorgaande versie van hetzelfde verkooppuntId."
+      },
+      "lat": {
```

Comment:
Why not use a `geometry` field of type `Point`? That would help geosearchability of the data.

Reply:
We decided to define these as string for now, since there seems to be no way to write geo data to postgres from spark.
Converting later on should be easy